docs/buildah-bud.md

```diff
@@ -287,7 +288,8 @@ those.
 
 **--iidfile** *ImageIDfile*
 
-Write the image ID to the file.
+Write the built image's ID to the file.  When `--platform` is specified more
+than once, only one of their IDs will be written to the file.
```

Comment:
Is it the first one, or random chance?

Reply:
Should be first, I would guess, coding might be easier to do last.  Other option would be to conflict them so that --iidfile can only be used if user specifies < 2 platforms.  (Which I think is the best solution.

Reply:
There's no coordination, so currently I expect it's the one that wrote (or overwrote) it most recently.

Reply:
Can we just block the option?

Reply:
Sure, changing it.